### DISCUSSION TITLE
Bulk Update Variant Pricing

### DISCUSSION THREAD

**Discussion by YoniSoli** [saleor/saleor#9509] [Q&A]
Hi, I am using Saleor and I have multiple channels per product and variant.
I need to update the pricing for a variant, and I need to do it for every channel.
Is there a way to bulk update every channel with the new pricing? 
Or some way that a variant's price will be taken from a single channel, that will be the "main" one?
I am using version 3.1.
Thanks!

**Comment by karolkielecki** [comment]
Hi @YoniSoli, 
You can update pricing in multiple channels simultaneously for one variant using [the following mutation](https://docs.saleor.io/docs/3.x/developer/api-reference/mutations/product-variant-channel-listing-update). But IMO, the best solution would be to create [an app](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts).
